src/Microsoft.OpenApi.OData.Reader/Common/Helpers.cs

```diff
@@ -0,0 +1,54 @@
+﻿using System.Collections.Generic;
+using System.Linq;
+using Microsoft.OData.Edm;
+using Microsoft.OpenApi.Models;
+
+namespace Microsoft.OpenApi.OData.Common
+{
+    internal static class Helpers
```

Comment:
more meaning full class name. for example "xxxxHelpers"

Reply:
Renamed appropriately